python/plugins/processing/gui/BatchInputSelectionPanel.py

```diff
@@ -86,13 +86,11 @@ def showPopupMenu(self):
 
         if not (isinstance(self.param, QgsProcessingParameterMultipleLayers) and
                 self.param.layerType == dataobjects.TYPE_FILE):
-            selectLayerAction = QAction(
-                QCoreApplication.translate('BatchInputSelectionPanel', 'Select from Open Layers…'), self.pushButton)
+            selectLayerAction = QAction(self.tr('Select from Open Layers…'), self.pushButton)
```

Comment:
Just wondering if we are not loosing translation with this change in the LTR?
Maybe keep this change for the master branch if we loose translation?

Reply:
If I'm not wrong, context remains the same, isn't it?